### DISCUSSION TITLE
Convert to CoreML

### DISCUSSION THREAD

**Discussion by ballplayajd** [ml-explore/mlx#218] [General]
Is there a way to convert MLX models directly to coreml? Didnt see a function for it but might have missed

**Comment by RahulBhalley** [comment]
Thumbs up for this one! Would be really really helpful.

**Comment by awni** [comment]
Not yet, but we can look into this.

**Comment by cchance27** [comment]
Was wondering this as well, as was trying to understand if MLX could interop with CoreML, was looking at some conversion code from the CoreML/PyTorch side but things like the jit tracing don't seem to be possible so not sure how it would be done :S
